collect_app/src/main/java/org/odk/collect/android/projects/ManualProjectCreatorDialog.kt

```diff
@@ -81,6 +81,7 @@ class ManualProjectCreatorDialog : MaterialFullScreenDialogFragment(), Duplicate
                 confirmationArgs.putString(MATCHING_PROJECT, uuid)
                 DialogUtils.showIfNotShowing(DuplicateProjectConfirmationDialog::class.java, confirmationArgs, childFragmentManager)
             } ?: run {
+                Analytics.logEvent(AnalyticsEvents.GOOGLE_ACCOUNT_PROJECT)
```

Comment:
This only gets the case where there is no existing matching project. I think we also need to track when there is an existing matching project and the user confirms creating a new one. In fact, I think we should track both cases.

Reply:
When I said "both cases", both confirming and switching projects after a duplicate project configuration attempt. At the time I was imagining we could do this separately for Google Drive and other manual configuration but that gets pretty unwieldy. So instead I tracked duplicate attempts and resolution by switching across all config methods.